src/js/flows/ingestFiles.js

```diff
@@ -51,16 +51,23 @@ const validateInput = (paths) => ({
 
 const createDir = () => ({
   async do({dataDir}) {
-    await fsExtra.ensureDir(dataDir)
+    dataDir && (await fsExtra.ensureDir(dataDir))
```

Comment:
It seems like this whole step will be removed once https://github.com/brimsec/brim/issues/676 is done. All we might do is check that the user specified directory exists.

Reply:
Wait, I might understand the data_dir thing correctly. This might be fine.